lib/Messenger/Handler/CleanupThumbnailsHandler.php

```diff
@@ -35,6 +35,7 @@ public function __invoke(CleanupThumbnailsMessage $message, Acknowledger $ack =
         return $this->handle($message, $ack);
     }
 
+    // @phpstan-ignore-next-line
```

Comment:
whats the problem here?

Reply:
It complains about the method not being used anywhere. Infact, It is used by messenger batch handler trait.